worlds/wargroove2/Items.py

```diff
@@ -0,0 +1,140 @@
+import typing
+
+from BaseClasses import Item, ItemClassification
+from typing import Dict, List
```

Comment:
```suggestion
from BaseClasses import Item, ItemClassification
from typing import Dict, List, NamedTuple, Optional
```
Would recommend putting them all in one import, then changing the uses of NamedTuple and Optional in the file